Test, Signup, Base forms

jlc1541 opened the pull request:
Contains a few labels to inform the reader on what information should be entered and where they should place it on the signup form.
Base.html has been changed to move the hyperlinks to the center of the screen and remove decorations.
Test.html has changed the words to "Welcome to PotatoMail" and was given a title attribute when hovering over it.

jlc1541 committed:
Add files via upload

haleyhagler227 commented on app/templates/base.html:
```diff
@@ -1,45 +1,23 @@
+        </ul>
+        {% endif %}
+        {% endwith %}
+        {% block content %}{% endblock %}
```
for compatibility with the rest of the app change content to app_content

haleyhagler227 commented on app/templates/base.html:
```diff
@@ -1,45 +1,23 @@
+    </head>
+    <body>
+    
+        <div style="margin: 50px;text-align: center; font-size: 30px;">
+                <a href="{{url_for('test') }}" style="text-decoration:none">Home</a>
+                <a href="{{url_for('login') }}"style="text-decoration:none">Login</a>
+                <a href="{{url_for('SignUp') }}"style="text-decoration:none">SignUp</a>
+        </div>
```
Change this back to block nav bar for compatibility with other parts of the app

haleyhagler227 commented on app/templates/base.html:
```diff
@@ -1,45 +1,23 @@
-{% endblock %}
-
-
-{% block navbar %}
-
-
-    <nav class="navbar navbar-default">
-        <div class="container-fluid">
-            <div class="navbar-header">
-                <a class="navbar-brand" >Potato Mail </a>
-            </div>
-            <div class="nav navbar-nav">
-                <li><a href="{{url_for('test') }}">Home</a></li>
-                <li><a href="{{url_for('login') }}">Login</a></li>
-                <li><a href="{{url_for('index') }}">Inbox</a></li>
-                
-                
-            </div>
-        </div>
-    </nav>
-{% endblock %}
```
This is the nav bar that should stay.

jlc1541 replied:
This makes a lot of sense to me. I've already seen this criticism so I understand that these changes are very significant to user experience.

haleyhagler227 commented on app/templates/signup.html:
```diff
@@ -0,0 +1,54 @@
+            {{ form.OHostName.label }}
+            {{ form.OHostName( size = 32)}}
+        </p>
+        <p>
+            {{ form.O_username.label }}
+            {{ form.O_username(size = 32)}}
+        </p>
+        <p>
+            {{ form.O_password.label }}
+            {{ form.O_password(size = 32)}}
+        </p>
```
username and password are the same for each

haleyhagler227 commented on app/templates/signup.html:
```diff
@@ -0,0 +1,54 @@
+            {{ form.OHostName.label }}
+            {{ form.OHostName( size = 32)}}
+        </p>
+        <p>
```
Add a OPortNum
```suggestion
        <p>
        {{ form.OPortNum.label }}
        {{ form.OPortNum(size = 32)}}
        </p>

```

haleyhagler227 commented on app/templates/signup.html:
```diff
@@ -0,0 +1,54 @@
+            {{ form.IHostName.label }}
+            {{ form.IHostName(size = 32)}}
+        </p>
+        <p>
+            {{ form.username.label }}<br>
+            {{ form.username(size=32) }}
+        </p>
+        <p>
+            {{ form.password.label }} <br>
+            {{ form.password(size=32) }}
+        </p>
```
Add a IPortNum
```suggestion
        <p>
        {{ form.IPortNum.label }}
        {{ form.IPortNum(size = 32)}}
        </p>

```

haleyhagler227 commented on app/templates/signup.html:
```diff
@@ -0,0 +1,54 @@
+{% extends "base.html" %}
+
```
Make a second file, named registerEmail.html, or anything kind of like that and put this content. Use Signup.html to get a username and a password (have two the password fields so they can be compared)

haleyhagler227 commented on app/templates/signup.html:
```diff
@@ -0,0 +1,54 @@
+            {{ form.email.label }}<br>
+            {{ form.email(size = 32)}}
+        </p>
+
```
put the password field here. And only ask for it once

haleyhagler227 commented on app/templates/signup.html:
```diff
@@ -0,0 +1,54 @@
+    <form action="" method="post" novalidate>
+        {{form.hidden_tag() }}
+        
+        <p>
+            {{ form.name.label }}<br>
+            {{ form.name(size = 32)}}
+        </p>
```
I would remove this

jlc1541 replied:
That is probably best considering we already have a username.

haleyhagler227 commented on app/templates/signup.html:
```diff
@@ -0,0 +1,54 @@
+        color: grey;
+        }
+    </style>
+    <h1> Register </h1>
```
Register Email

jlc1541 replied:
Got you :)

haleyhagler227 reviewed (changes requested):
Main point is signup looks like it is trying to play two functions: signing people up to the site and registering emails. I think it would be better to separate the two. So the email registration can be reused at other points in the app if necessary.

jlc1541 committed:
Update signup.html

jlc1541 committed:
Update base.html

jlc1541 reviewed (commented):
I agree with you on this one. It would be better for the fluidity of the user experience to keep that nav bar. Nice Suggestion.

jlc1541 committed:
Update signup.html

jlc1541 committed:
Update base.html

jlc1541 committed:
Update base.html

jlc1541 committed:
Update app/templates/signup.html

Co-authored-by: haleyhagler227 <70032753+haleyhagler227@users.noreply.github.com>

jlc1541 committed:
Update app/templates/signup.html

Co-authored-by: haleyhagler227 <70032753+haleyhagler227@users.noreply.github.com>

jlc1541 committed:
Update signup.html

Username and password have been modified to only ask once, and port numbers have been added as well.

jlc1541 committed:
Update base.html

Navbar integrated.

jlc1541 committed:
Merge branch 'master' into jlc1541--New-forms-and-edit-to-htmls
